.github/ISSUE_TEMPLATE/undergrad-research.md

```diff
@@ -19,6 +21,8 @@ We are thrilled that you're interested in research in our research group! Please
 
 **Relevant classes**:
 
+**Willing to do research during the summer:**
```

Comment:
We kinda already ask this below under "When do you want to do research?" Maybe we can clarify there that people should say, even if they want to get started during the semester, that they should answer whether they're interested in _continuing_ into the summer.

Reply:
Addressed